src/pages/api/auth/[...nextauth].ts

```diff
@@ -6,6 +6,15 @@ import { SendVerificationRequestParams } from "next-auth/providers";
 import { email as sendEmail, emailRoleIgnoreError } from "../../../api/sendgrid";
 import randomNumber from "random-number-csprng";
 import { toChinese } from "shared/strings";
+import { userAttributes } from "api/database/models/attributesAndIncludes";
+import invariant from "tiny-invariant";
+import User from "api/database/models/User";
+
+declare module "next-auth" {
```

Comment:
can you add a comment explaining **why** we need this block? Note that it's not about **what** this block does.